src/Model/Model.java

```diff
@@ -535,6 +545,68 @@ public boolean sendMessage(Event data) {
         return true;
     }
 
+    public ClaimGrayResult claimGray(ClaimGrayRequest req){
+        ClaimGrayResult res = new ClaimGrayResult();
+        String username = req.getUsername();
+        Game game = getAssociatedGame(username);
+        if (game.claimRoute(username, req.getId())) {
```

Comment:
I thought the TAs said we had to let the user choose what color to use when it's gray. This would just assign a color to the route.